docs/en/api/S3-API.md

```diff
@@ -30,6 +30,10 @@ Only top-level Alluxio directories are treated as buckets by the S3 API.
 
 Alluxio uses `/` as a reserved separator. Therefore, any S3 paths with objects or folders named `/`
 (eg: `s3://example-bucket//`) will cause undefined behavior.
+There are some special characters and patterns in file names are not supported, please avoid creating object key names with these patterns, or acquire additional encoding from client end.
```

Comment:
This looks like is generally not supported in Alluxio URI other than S3-API, e.g. FS API and POSIX API has the same problem

Reply:
is there an central hub calling out file names limitations I could put the wording there? or I need to add on both FS API and POSIX API page?

Reply:
try to find the common place in doc but could not find it
Unified-Namespace.md doc could be a place to put it